code/modules/awaymissions/corpse.dm

```diff
@@ -406,7 +406,7 @@
 /obj/effect/mob_spawn/human/nanotrasensoldier
```

Comment:
These are spawned by the Orion Trail arcade emag setting, being a minor buff to emags.